samples/csharp/getting-started/BinaryClassification_AutoML/README.md

```diff
@@ -0,0 +1,72 @@
+# Sentiment Analysis for User Reviews
+
+## Automated Machine Learning
+Automated machine learning (AutoML) automates the end-to-end process of applying machine learning to real-world problems. Given a dataset, AutoML iterates over different data featurizations, machine learning algorithms, hyperparamters, etc. to select the best model based on training scores.
+
+## Problem
+This problem is to predict if a customer's review has a positive or negative sentiment. We will use small wikipedia-detox-datasets: one dataset for training and a second to test the model produced by AutoML. Human judges assigned every comment in these datasets a sentiment label:
```

Comment:
```suggestion
This problem is to predict if a Wikipedia comment contains a personal attack or not. We will use small sample of the Wikipedia Detox dataset: one dataset for training and a second to test the model produced by AutoML. Human judges assigned every comment in these datasets a toxicity label:
```

Reply:
thank you -- this was directly copied from https://github.com/dotnet/machinelearning-samples/blob/master/samples/csharp/getting-started/BinaryClassification_SentimentAnalysis/README.md , then fixed up a bit. thank you for the further fix up